### ISSUE TITLE
Platinum - Range Queries with Sweep Line

### ISSUE THREAD

**Issue by bqi343** [cpinitiative/usaco-guide#487]
https://usaco.guide/plat/range-sweep

> the most important part,the tutorial is missing

**Comment by dutinmeow** [comment]
there's really not much to say since range sweep is such a broad topic

**Comment by timothygao8710** [comment]
I agree. If we were to do a tutorial it would probably be most helpful if it was regarding a problem representative of "Range Queries with Sweep Line."
